src/cmd-import

```diff
@@ -155,6 +155,10 @@ def generate_build_meta(tmp_oci_archive, tmp_oci_manifest, metadata, ostree_comm
             'commit': commit,
         }
 
+    summary = metadata.get('Labels', {}).get('org.opencontainers.image.title')
```

Comment:
since we populate `description` too I think it seems more obvious if we were to use that here instead of `title`.

```suggestion
    summary = metadata.get('Labels', {}).get('org.opencontainers.image.description')
```

optional, of course.